docs: add caution box to cluster config command (#670)
redpanda-data/documentation: modules/manage/pages/kubernetes/security/tls/k-cert-manager.adoc

--
======
+
[NOTE]
====
include::manage:partial$kubernetes/tls-update-note.adoc[]
====

. Make sure the Certificates are in a `READY` state.
+
```bash
kubectl get certificate --namespace <namespace>
```

> By default, this certificate will be used to encrypt traffic between clients and all external listeners. You can select specific certificates for each external listener. See xref:manage:kubernetes/networking/k-configure-listeners.adoc#tls[Configure Listeners in Kubernetes].

Reviewer: ```suggestion
By default, this certificate is used to encrypt traffic between clients and all external listeners. You can select specific certificates for each external listener. See xref:manage:kubernetes/networking/k-configure-listeners.adoc#tls[Configure Listeners in Kubernetes].
```
